src/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/Components/ArtworkDetailsForm.tsx

```diff
@@ -206,6 +218,8 @@ export const ArtworkDetailsForm: React.FC = () => {
       values.location.countryCode?.toUpperCase()
     )
 
+  const categories = useRef(acceptableCategoriesForSubmission()).current
```

Comment:
**suggestion:** I guess, we can use `useMemo` here because the value of categories doesn't have to be mutable ([context](https://stackoverflow.com/questions/66429202/what-are-production-use-cases-for-the-useref-usememo-usecallback-hooks#:~:text=useRef%20can%20be%20used%20to,concept%20wise)%20and%20skip%20recalculation.)).

```suggestion
  const categories = useMemo(acceptableCategoriesForSubmission()).current
```

Reply:
@olerichter00 cool, that seems better. 👍 . Will update and test.

Reply:
Thanks